File: Infernus/InfernusDAO/Models/Armadura.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace InfernusDAO
{
    public class Armadura
    {
        private string Item;
        private string Nome;
        private string Tipo_armadura;
        private string Custo;
        private string Moeda;
        private string Ca;
        private string Furtv;
        private string Forca;
        private string Peso;
        private string Durabilidade;
        private string Observacao;

        public string item
        {
            get { return Item; }
            set { Item = value; }
        }

        public string nome
        {
            get { return Nome; }
            set { Nome = value; }
        }

        public string custo
        {
            get { return Custo; }
            set { Custo = value; }
        }

        public string moeda
        {
            get { return Moeda; }
            set { Moeda = value; }
        }

        public string tipoArmadura
        {
            get { return Tipo_armadura; }
            set { Tipo_armadura = value; }
        }

        public string ca
        {
            get { return Ca; }
            set { Ca = value; }
        }

        public string peso
        {
            get { return Peso; }
            set { Peso = value; }
        }

        public string forca
        {
            get { return Forca; }
            set { Forca = value; }
        }

        public string durabilidade
        {
            get { return Durabilidade; }
            set { Durabilidade = value; }
        }

        public string furtv
        {
            get { return Furtv; }
            set { Furtv = value; }
        }

        public string observacao
        {
            get { return Observacao; }
            set { Observacao = value; }
        }
    }
}

File: BrincadeirasJava/src/infernusJava/models/Armadura.java
package infernusJava.models;

public class Armadura {

	private int id;
	private String nome;
	private String tipo;
	private String custo;
	private String ca;
	private String forca;
	private String furtv;
	private String peso;
	private String observacao;

	public Armadura() {

	}

	public int getId() {
		return id;
	}

	public void setId(int id) {
		this.id = id;
	}

	public String getNome() {
		return nome;
	}

	public void setNome(String nome) {
		this.nome = nome;
	}

	public String getTipo() {
		return tipo;
	}

	public void setTipo(String tipo) {
		this.tipo = tipo;
	}

	public String getCusto() {
		return custo;
	}

	public void setCusto(String custo) {
		this.custo = custo;
	}

	public String getCa() {
		return ca;
	}

	public void setCa(String ca) {
		this.ca = ca;
	}

	public String getForca() {
		return forca;
	}

	public void setForca(String forca) {
		this.forca = forca;
	}

	public String getFurtv() {
		return furtv;
	}

	public void setFurtv(String furtv) {
		this.furtv = furtv;
	}

	public String getPeso() {
		return peso;
	}

	public void setPeso(String peso) {
		this.peso = peso;
	}

	public String getObservacao() {
		return observacao;
	}

	public void setObservacao(String observacao) {
		this.observacao = observacao;
	}

	@Override
	public String toString() {
		return "Armadura [id=" + id + ", nome=" + nome + ", tipo=" + tipo + ", custo=" + custo + ", ca=" + ca
				+ ", forca=" + forca + ", furtv=" + furtv + ", peso=" + peso + ", observacao=" + observacao + "]";
	}

	public Armadura(int id, String nome, String tipo, String custo, String ca, String forca, String furtv, String peso,
			String observacao) {
		super();
		this.id = id;
		this.nome = nome;
		this.tipo = tipo;
		this.custo = custo;
		this.ca = ca;
		this.forca = forca;
		this.furtv = furtv;
		this.peso = peso;
		this.observacao = observacao;
	}

}
